app/src/main/java/com/kickstarter/viewmodels/BackingAddOnViewHolderViewModel.kt

```diff
@@ -249,6 +249,9 @@ class BackingAddOnViewHolderViewModel {
             this.quantity
                     .compose<Pair<Int, Reward>>(combineLatestPair(addOn))
                     .map { data -> Pair(data.first, data.second.id()) }
+                    .distinctUntilChanged { item1, item2 ->
```

Comment:
Could we rename `item1` and `item2` to be what the actual items are? Just to make it more clear.

Reply:
in this case item type is Pair(int, long), not sure what name give them for it to be clearer, what do you think it would be helpful here @leighdouglas ?